websites/S/Sushi Scan/presence.ts

```diff
@@ -0,0 +1,130 @@
+const presence = new Presence({
+		clientId: "1324071536017149973",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	_Logo = "1324137722079875112",
+	_Browsing = "1324141175866916864",
+	_Reading = "1324141174742581248",
+	_Selecting = "1324141175808196731",
+}
```

Comment:
Use image URLs rather than Discord asset names. For the asset names themselves, you can omit the `_`

Reply:
Used "Sushi" prefix to avoid conflict with the default Reading asset